service/utility.go

```diff
@@ -101,16 +103,31 @@ func generateUUID() (string, error) {
 	return strings.ToLower(id.String()), nil
 }
 
-// hashPassword hashes and salts provided string
+// TODO probably need to synchronize this as well
```

Comment:
need an issue to track this so we do not forget

Reply:
https://github.com/hwsc-org/hwsc-user-svc/issues/24